vtex-sites/base.store
# feat: Add Postal Code Input Component

## What's the purpose of this pull request?
This PR adds an input field to set user postal code, that will be used for the regionalization feature.

## How does it work?
The user types a postal code and presses enter, then the postal code is saved into the indexed db.

### Possibilities
- Put the Region component as subpart of Search Components folder
- Put the useRegion hook inside the Search SDK folder

## How to test it?
1. Open the preview
2. Click on Postal Code Input, type a postal code, then press Enter.
3. Check if the IndexedDB Updated with the new value

### References
- Set font-display: optional to fix CLS issues (https://deploy-preview-15--upbeat-shirley-608546.netlify.app/posts/high-performance-web-font-loading/)
- https://www.smashingmagazine.com/2021/05/reduce-font-loading-impact-css-descriptors/

## Checklist

<em>You may erase this after checking them all ;)</em>
